subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/artifactreuse/ResolutionOverrideIntegrationTest.groovy

```diff
@@ -16,8 +16,13 @@
 package org.gradle.integtests.resolve.artifactreuse
 
 import org.gradle.integtests.fixtures.AbstractHttpDependencyResolutionTest
+import spock.lang.Shared
 
 class ResolutionOverrideIntegrationTest extends AbstractHttpDependencyResolutionTest {
+
+    @Shared
+    def refreshDependenciesArgs = ['-U', '--refresh-dependencies']
```

Comment:
@ljacomet Are we ok with such "deduplication"? Or we would prefer just copy/paste string literals, considering the fact that very unlikely we will have another additional option in future(what will force us to make change it 3 places instead of one)

Reply:
I find it good practice to extract this as you did, so 👍 from me